.changeset/green-moose-leave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-test-utils': patch
+---
+
+Backends started with `startTestBackend` are now automatically stopped after the each test, unless the `autoStop` option is set to `false`.
```

Comment:
Actually `'never'` below, but tbh `false` probably feels better 🤔  Unless one wants to stay consistently with strings